fashion_ai_bounded_autonomy/performance_tracker.py

```
@@ -295,8 +352,24 @@ async def update_proposal_status(
-        """Get KPI summary for specified period"""
+    async def get_kpi_summary(self, days: int = 7) -> dict[str, Any]:
+        """
+        Summarize per-agent KPIs over the most recent time window.
+        
+        Produces per-agent statistics computed from the 'execution_time' entries in the agent_metrics table for the past `days` days. The agent KPIs include:
+        - `active_days`: number of distinct days the agent reported execution_time,
+        - `total_operations`: total execution_time samples,
+        - `avg_execution_time`: average execution_time for the period.
+        
+        Parameters:
+        	days (int): Number of days to include in the summary window (default 7).
```

Flake8 (7.3.0)

[error] indentation contains mixed spaces and tabs

(E101)